gnat2goto/ireps/ireps_generator.py

```diff
@@ -28,22 +28,37 @@
 
 import os
 import os.path
-import sys
 import json
 import argparse
 
 from pprint import pprint
 from glob import glob
 from copy import copy
 
+
+# Helper functions
+
 irep_type_to_ada_type = {
-   "irep"      : "Irep",
-   "bool"      : "Boolean",
-   "integer"   : "Integer",
-   "string"    : "String",
-   "gnat:sloc" : "Source_Ptr"
+    "irep"      : "Irep",
+    "bool"      : "Boolean",
+    "integer"   : "Integer",
+    "string"    : "String",
+    "gnat:sloc" : "Source_Ptr"
 }
 
+special_names = {"+"      : "op_add",
```

Comment:
This appears to be a map of operator symbols to operator names. Can it get appropriately renamed?

Reply:
It's not that I cannot. It's more of not changing things that would cause the original author to find them difficult to change, since I don't know if he still bothers maintaining the file. I could have made more changes but I'm sticking to the minimum.

Reply:
I wouldn't worry about that; We're 'owning' this repository now. In any case, a large-ish refactoring would make any changesets based on the older version invalid most of the time anyway.